exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/SetOptionHandler.java

```diff
@@ -63,9 +64,10 @@ public final PhysicalPlan getPlan(SqlNode sqlNode) throws ForemanSetupException
     SqlNode optionValue = statement.getValue();
 
     if (optionValue == null) {
+      OptionDefinition optionDefinition = optionManager.getOptionDefinition(optionName);
```

Comment:
Probably worth a comment saying that this call will validate the option to ensure it is valid.

Reply:
Thanks, added a comment.